apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -601,6 +618,16 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
                     this.parseBalRes(res);
                 });
             });
+
+        this.voucherDetails$.subscribe(response => {
+            if (response) {
```

Comment:
Add nullity check to response.subTotal or based on status 'success'

Reply:
there is no any success status in response only voucher object is there in response